src/openzaak/components/catalogi/admin/zaaktypen.py

```diff
@@ -331,15 +331,18 @@ def _publish_validation_errors(self, obj):
         ):
             errors.append(_("All related resources should be published"))
 
-        form = self.form(instance=obj, data={**model_to_dict(obj), "_publish": "1"})
+        form = self.form(
```

Comment:
This method is not used anymore in ZaakTypeAdmin, because it is called in `response_post_save_change` which is used then the form is submitted. But `publish` is not a submit action for ZaakType now. I think the whole method can be deleted

Reply:
I have removed the form, but it is still used by the admin publish action